src/baldr/incident_singleton.h

```diff
@@ -195,23 +196,23 @@ struct incident_singleton_t {
     LOG_INFO("Incident watcher started");
     // try to configure for changelog mode
     std::unique_ptr<valhalla::midgard::sequence<uint64_t>> changelog;
-    filesystem::path inc_log_path(config.get<std::string>("incident_log", ""));
-    filesystem::path inc_dir;
+    std::filesystem::path inc_log_path(config.get<std::string>("incident_log", ""));
+    std::filesystem::path inc_dir;
+    bool inc_dir_exists = false;
```

Comment:
just a little refactor, bcs `std::filesystem::path` can't be initialized like it's used here.